x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -79,6 +84,17 @@ export class JobCreator {
     this._datafeed_config.query = query;
   }
 
+  protected updateModelMemoryEstimation() {
```

Comment:
for consistency this should start with an underscore

Reply:
https://github.com/elastic/kibana/pull/60888/commits/bdfaeb489a6ab879c8212a804155cfc6d54bdbe8